projects/ion/src/lib/switch/switch.component.html

```diff
@@ -0,0 +1,13 @@
+<button
+  [id]="key"
+  data-testid="ion-switch"
+  class="ion-switch"
+  [attr.name]="key"
+  [disabled]="disabled"
+  [class.ion-switch--active]="value"
+  [class.ion-switch--inactive]="!value"
+  [attr.size]="size"
+  [disabled]="disabled"
+  (click)="handleClick(value)">
+  <ng-content></ng-content>
```

Comment:
I don't understand why this is here, once this type of component don't need receive content inside on it.

Reply:
lint keeps complaining about the button having no content, maybe there are better alternatives, we can think about them

 https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin-template/docs/rules/elements-content.md